### PR TITLE
increase container pay

### PR DESCRIPTION
**Pull request by Crownapple**
I believe should increase due to new job in the city that pay double if not triple this job

### CODE REVIEW

**Comment by Zeeroqt** [comment]
400 sounds reasonable 500 is pushing it

**Comment by Crownapple** [comment]
> 400 sounds reasonable 500 is pushing it

I think you need to read the whole thing before you start commenting on anything.

**Comment by minipunch** [comment]
> new job in the city that pay double if not triple this job

What job u referring to ?

**Comment by Crownapple** [comment]
> > new job in the city that pay double if not triple this job
> 
> What job u referring to ?

construction you make about $900^ pre point and it takes about 5 seconds or so and for doc job/ container you make about anyway for 800 to 1.3k and takes around anyway from 35 secs to a min or so depending on the where the container is located

File: resources/ContainerForklift/server.lua [hunk 1]
```diff
@@ -94,10 +94,10 @@ AddEventHandler("containerjob:reward", function(securityToken)
         targetContainer = onJob[source]
         -- calculate reward based on distance from drop off
         local dist = #(vector3(targetContainer.x, targetContainer.y, targetContainer.z) - vector3(CONTAINER_DROP_OFF.x, CONTAINER_DROP_OFF.y, CONTAINER_DROP_OFF.z))
-        local reward = math.floor(dist * 3.5)
+        local reward = math.floor(dist * 4.5)
```
Comment (minipunch) [inline resources/ContainerForklift/server.lua:97]:
```suggestion
        local reward = math.floor(dist * 4.15)
```

File: resources/ContainerForklift/server.lua [hunk 2]
```diff
@@ -94,10 +94,10 @@ AddEventHandler("containerjob:reward", function(securityToken)
...
         -- give reward
         local char = exports["usa-characters"]:GetCharacter(source)
-        char.giveBank(reward + math.random(0, 250), "Container Reward")
+        char.giveBank(reward + math.random(0, 500), "Container Reward")
```
Comment (minipunch) [inline resources/ContainerForklift/server.lua:100]:
```suggestion
        reward = reward + math.random(0, 500)
        char.giveBank(reward, "Container Reward")
```

**Review by minipunch — APPROVED** [review]

**Pull request merged**